table_size.go

```diff
@@ -15,8 +15,12 @@ const rowCountCutoff = 10000.0
 // NewTableSizeCollector returns a new collector that monitors table sizes.
 func NewTableSizeCollector(namespace string, db *sql.DB) (*dbTableSizeCollector, error) {
 	var dbName string
-	q := `SELECT current_database();`
-	err := db.QueryRow(q).Scan(&dbName)
+	err := db.QueryRow(`SELECT current_database()`).Scan(&dbName)
+	if err != nil {
+		return nil, err
+	}
+	var schemaName string
+	err = db.QueryRow(`SELECT current_schema()`).Scan(&schemaName)
```

Comment:
current_schema can return null if the search path is empty [1]. This should handle that (default to public probably).

[1]: https://www.postgresql.org/docs/10/static/functions-info.html